apps/api/src/app/blueprint/blueprint.module.ts

```diff
@@ -1,12 +1,12 @@
 import { MiddlewareConsumer, Module, NestModule } from '@nestjs/common';
 
 import { SharedModule } from '../shared/shared.module';
-import { WorkflowModule } from '../workflows/workflow.module';
 import { USE_CASES } from './usecases';
 import { BlueprintController } from './blueprint.controller';
+import { WorkflowModuleV1Deprecated } from '../workflows-v1-deprecated/workflowModuleV1Deprecated';
 
 @Module({
-  imports: [SharedModule, WorkflowModule],
+  imports: [SharedModule, WorkflowModuleV1Deprecated],
```

Comment:
V1 is not deprecated. It's just V1.

```suggestion
  imports: [SharedModule, WorkflowModuleV1],
```